.github/workflows/tests.yml

```diff
@@ -35,6 +35,8 @@ jobs:
             symfony-version: "^7.1"
           - php-version: 8.4
             symfony-version: "^7.1"
+          - php-version: 8.4
```

Comment:
Please remove this and update `7.1` to `7.2` on the previous lines.

Reply:
Updated